app/navigation/houseConsignment/index/items/HouseConsignmentItemNavigator.scala

```diff
@@ -200,13 +211,58 @@ class HouseConsignmentItemNavigator(houseConsignmentMode: Mode) extends Navigato
             controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex)
         }
     }
+
+  private def netWeightPageRoute(ua: UserAnswers, houseConsignmentMode: Mode, itemMode: Mode, houseConsignmentIndex: Index, itemIndex: Index): Option[Call] =
+    (config.phase6Enabled, houseConsignmentMode, itemMode) match {
+      case (true, _, NormalMode) =>
+        Some(routes.UniqueConsignmentReferenceYesNoController.onPageLoad(ua.id, houseConsignmentIndex, itemIndex, houseConsignmentMode))
+      case (false, _, NormalMode) =>
+        Some(routes.AddCustomsUnionAndStatisticsCodeYesNoController.onPageLoad(ua.id, houseConsignmentIndex, itemIndex, houseConsignmentMode, itemMode))
+      case (_, CheckMode, CheckMode) => Some(controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex))
```

Comment:
Very minor but `houseConsignmentMode` is pretty obsolete in the pattern match here

Could perhaps do

```
itemMode match {
  case NormalMode if config.phase6Enabled => ???
  case NormalMode => ???
  case CheckMode => ???
}
```